docs/features.rst

```diff
@@ -324,6 +324,26 @@ instead of deleting it completely. To do so, one can set the directive::
 By default all leaves have it set to ``true`` (such node is selected)
 and branches have set it to ``false`` (such node is not selected).
 
+.. _sort:
+
+Sort
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This document seems to be focused on the fmf file interface and currently this part is kinda out of place. Maybe we could add a `/.sort` (similar to `select`) interface. Otherwise I think having a python interface documentation would be better.

Reply:
Yeah, you're right that this new content is more related to Python coding. We already have the `Modules` section. Perhaps we can rename it to `Code` to have there all information related to Python code in a similar way as we have it for `tmt`. I've moved the new section there in c89892c. Will do the rename and cleanup in a separate pull request.

Reply:
I think it is more standard to call it `API Reference` or `Python (API) Reference`, but that's for another issue.